common/flatpak-dir.c

```diff
@@ -6569,6 +6606,9 @@ flatpak_dir_uninstall (FlatpakDir          *self,
                    _("%s branch %s is not installed"), name, parts[3]);
     }
 
+  if (!log_uninstall_event (self, ref, cancellable, &logging_error))
+    g_warning ("Failed to log uninstallation: %s", logging_error->message);
```

Comment:
Missing `g_clear_error (&logging_error)`. (I realise the function returns immediately after this, but it would safeguard against double-setting the error if the function were to ever be extended.)